internal/driver/glfw/window.go

```diff
@@ -540,9 +540,11 @@ func (w *window) processMouseClicked(button desktop.MouseButton, action action,
 	if wid, ok := co.(fyne.Focusable); !ok || wid != w.canvas.Focused() {
 		ignore := false
 		_, _, _ = w.findObjectAtPositionMatching(w.canvas, mousePos, func(object fyne.CanvasObject) bool {
-			switch object.(type) {
-			case fyne.Focusable:
+			if focusable, isFocusable := object.(fyne.Focusable); isFocusable {
 				ignore = true
+				if focusable != w.canvas.Focused() {
+					w.canvas.Focus(focusable)
```

Comment:
No, we have specifically set it so that a focusable item does not automatically get focus when tapped - the objects would have to request it.